plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/groovy/GroovyScript/help-pattern.html

```diff
@@ -1 +0,0 @@
-Note that due to security reasons scanning the console log using a Groovy script is not allowed.
```

Comment:
Rather than deleting this file it would make sense to copy the help from the other `help-pattern.html` file.

Reply:
Ideally, I'd like the other file to work for both cases 😉 
However, if I can't make it do that then, yes, I'll make a GroovyScript-specific version here.

Reply:
I tried that but it seems that Jenkins core does not support inheritance on those files. So I ended up copying the same files to all the folders.

Reply:
I've experimented and I've found that, with the GroovyScript's help-pattern.html gone, Jenkins does inherit.
So I've made the inherited one flexible and able to cope with all 3 scenarios (default pattern, optional pattern, mandatory pattern) and I've checked that this works on e.g. both GroovyScript, CCM and others.

I did find that if you define a config.jelly file then that'll be used _instead_ of the parent's config.jelly but it's able to reference help files in the parent without a problem.